src/main/java/billennium/tests/service/question/QuestionService.java

```diff
@@ -4,7 +4,9 @@
 import billennium.tests.model.QuizModel;
 
 import java.util.List;
-
+// TODO do wszystkich serwisów jestem zwolennikiem nie śmiecenia kodu interfejsami do serwisu jezeli serwis ma tylko
+//  jedna implementacje.
+//  Możesz doczytać jak chcesz dlaczego kiedyś się tak robiło a teraz już raczej nie trzeba :)
```

Comment:
Ja jestem zwolennikiem zawsze interfejsu :) co jeśli kiedyś będzie chcieli to rozszerzyć?

Reply:
No i to raczej zboczennie z DDD wszędzie wszędzie Interfejs

Reply:
Wyjdźmy z założenia że jak będzie serwis miał dwie różne implementacje to dopisujemy interfejs. Ogólnie pisanie interfejsu do każdego serwisu to stary styl pisania. Wg mnie 95% serwisu w tej apce będzie mało jedną implementację i pisanie interfejsu do każdego zasyfi kod. Przypomnij mi kiedyś to opowiem Ci fajną rzecz o interfejsach w serwisach.